packages/email-components/src/scss/foundation/_colors.scss

```diff
@@ -1,5 +1,5 @@
 @use '@tokens' as tokens;
 
-.Page {
+.background-cover {
```

Comment:
Along with conventions from Spirit DS the class name should be `.bg-cover` and it should be one of the utilities, not a color within foundation directory.